Assets/Scripts/MessageImageDisplayer.cs

```diff
@@ -425,7 +427,7 @@ public void SetDeliverItemText(string name)
             
         }
         Button btn = deliver_item_visual_dislay.GetComponent<Button>();
-        deliver_item_display_text.text = update_name;
+        deliver_item_display_text.text = LanguageSource.current_language.Equals(LanguageSource.LANGUAGE.ENGLISH) ? update_name : name;
```

Comment:
Why is this only for english? Should this just use "name" always?

Reply:
Freiberg asked me to capitalize their items.

Reply:
I would change "update_name" to "lowercase_name".
I would change it to check for german instead (switch the ternary statement) in case we add another language.
and Finally I would just put a small comment saying that "Freiburg wanted their words to be all uppercase".